src/combobox/ComboBox.ts

```diff
@@ -266,118 +267,158 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 			return (current + total) % total;
 		}
 
-		let index = nextIndex(this._activeIndex);
-
-		while (this._isIndexDisabled(index)) {
-			index = nextIndex(index);
-		}
-
-		this._activeIndex = index;
+		this._activeIndex = nextIndex(this._activeIndex);
 		this.invalidate();
 	}
 
-	protected onElementCreated(element: HTMLElement, key: string) {
+	protected onElementUpdated(element: HTMLElement, key: string) {
```

Comment:
Can we remove reliance on `onElement*` methods? Technically they're sort of kind of deprecated, though the message was temporarily removed. I'm wondering if we can leverage meta in a way that negates the need for these altogether.

Reply:
Would the animation meta be able to handle scroll? I suppose we could write our own within Listbox, but I'd rather use something that exists, if we can.

Reply:
Moved to meta 👍